### PR TITLE
Add battlepass and org changes

### PR DESCRIPTION
**Pull request by vovacha**
- battlepass changes
- org changes
- add hasura metadata
- add dev docker setup

### CODE REVIEW

File: dev/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      IPFS_API_URL: https://ipfs.infura.io:5001/api/v0
+      IPFS_CLIENT_ID: ""
+      IPFS_CLIENT_SECRET: ""
+      SERVER_PORT: 4000
+      CHAIN_RPC_URL: wss://node.dev.sub.zero.io/
+      CHAIN_RPC_URL_OVERRIDE: wss://node.dev.sub.zero.io/
+      DATABASE_URL: postgres://postgres:postgres@db:5432/graph-gamedao?schema=public
+
+  graph:
+    image: hasura/graphql-engine:v2.12.0
```
Comment (2075) [inline dev/docker-compose.yml]: @vovacha this needs a bump to v2.12.1 as this image does not exist anymore due to security issues (reason for the aws escalation)

**Review by 2075 — APPROVED** [review]
lgtm

File: dev/docker-compose.yml (hunk 1 above)
Comment (vovacha) [reply dev/docker-compose.yml]: This one is only used for local development, but I'll also update it. You are right.
I'll update gitlab repo later. And this version change will be one of the changes.

**Comment by vovacha** [comment]
I'll merge this later today after the proper testing on the dev.

**Pull request merged**